includes/general.php

```diff
@@ -532,7 +532,7 @@ function pods_deprecated( $function, $version, $replacement = null ) {
 function pods_help( $text, $url = null ) {
 
 	if ( ! wp_script_is( 'jquery-qtip2', 'registered' ) ) {
-		wp_register_script( 'jquery-qtip2', PODS_URL . 'ui/js/jquery.qtip.min.js', array( 'jquery' ), '2.2' );
+		wp_register_script( 'jquery-qtip2', PODS_URL . 'ui/js/qtip/jquery.qtip.min.js', array( 'jquery' ), '3.0.3' );
```

Comment:
* :warning: In footer ($in_footer) is not set explicitly wp_register_script; It is recommended to load scripts in the footer. Please set this value to `true` to load it in the footer, or explicitly `false` if it should be loaded in the header.
* :no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 14a540264975d6c6e5c6c69d0c661d7a</sub>